yarn.lock

```diff
@@ -2,3 +2,7 @@
 # yarn lockfile v1
 
 
+ember@^1.0.3:
```

Comment:
why is there a change in the yarn.lock?

it's not related to rails/ruby upgrade, is it ?